src/test/java/pl/project13/maven/git/GitPropertiesFileTest.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * This file is part of git-commit-id-plugin by Konrad 'ktoso' Malawski <konrad.malawski@java.pl>
+ *
+ * git-commit-id-plugin is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * git-commit-id-plugin is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with git-commit-id-plugin.  If not, see <http://www.gnu.org/licenses/>.
+ */	
+
+package pl.project13.maven.git;
+
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.util.FileUtils;
+import org.junit.Test;
+
+import java.io.File;
+import java.util.*;
+import static java.util.Arrays.asList;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+
+import static org.fest.assertions.Assertions.assertThat;
+import org.junit.runner.RunWith;
+import static org.mockito.internal.util.reflection.Whitebox.setInternalState;
+import pl.project13.maven.git.util.PropertyManager;
+
+@RunWith(JUnitParamsRunner.class)
+public class GitPropertiesFileTest extends GitIntegrationTest {
+
+  static final boolean UseJGit = false;
+  static final boolean UseNativeGit = true;
```

Comment:
Scala case style in Java app? ;) Please use Java style constants here. USE_JGIT, etc